Makefile

```diff
@@ -56,48 +56,50 @@ TEST_LDFLAGS=-ldflags "-X github.com/cilium/cilium/pkg/kvstore.consulDummyAddres
 TEST_UNITTEST_LDFLAGS=-ldflags "-X github.com/cilium/cilium/pkg/datapath.DatapathSHA256=e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
 
 define generate_k8s_api
-	cd "./vendor/k8s.io/code-generator" && \
-	GO111MODULE=off bash ./generate-groups.sh $(1) \
+	$(QUIET) cd "./vendor/k8s.io/code-generator" && \
```

Comment:
I think that we end up potentially duplicating these `$(QUIET)` statements if we have them inside the templates like this. For instance

```
	$(QUIET) $(call generate_k8s_api_deepcopy_deepequal_client,client,github.com/cilium/cilium/pkg/k8s/slim/k8s/api,"$\ ...
```
=>
```
$(call generate_k8s_api,deepcopy,github.com/cilium/cilium/pkg/k8s/client,$(1),$(2),$(3))
```
=>
```
$(QUIET) cd "./vendor/k8s.io/code-generator" && \ ...
```

Maybe we should only use those statements in the main targets and not inside the templates?

Reply:
hey joe, thanks for your input!

over the whole PR - they ended up being only in the template - and not in both (main target and template) :smile:

but would be definitely nicer to have them only in the main targets. it's just that this only works as intended if the template only contains one command. otherwise (e.g. `generate_k8s_protobuf` (where tools are getting installed first)) only the first commands gets silenced.

-> i moved the `$(QUIET)`'s to the main target and chained the installation commands with the generation command in the mentioned case!